DependencyInjection/Compiler/CsrfExtensionPass.php

```diff
@@ -29,6 +29,17 @@ public function process(ContainerBuilder $container)
                 $tokenStorageReference = new Reference('security.context');
             }
             $definition->replaceArgument(0, $tokenStorageReference);
+
+            // Symfony <2.8 BC
+            $typeExtensionTag = $definition->getTag('form.type_extension');
+            if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+                $typeExtensionTag['extended-type'] = 'Symfony\Component\Form\Extension\Core\Type\FormType';
+            } else {
+                $typeExtensionTag['alias'] = 'form';
+            }
+            $definition->setTags(array_merge($definition->getTags(), array(
+                'form.type_extension' => $typeExtensionTag,
+            )));
```

Comment:
This is wrong. You can have multiple tags with the same name.

IMO, the simpler solution would be to remove the tag entirely from the XML file and add it here rather than trying to edit it here.